packages/web3torrent/src/components/torrent-info/channels-list/ChannelsList.tsx

```diff
@@ -42,19 +42,19 @@ class ChannelsList extends React.Component<UploadInfoProps> {
     );
 
     const uploaded = wire ? wire.uploaded : 0;
-    const peerAccount = wire ? wire.paidStreamingExtension.peerAccount : 'unknown';
+    const peerAccount =
+      pseType === 'leecher' ? channels[channelId].beneficiary : channels[channelId].payer;
```

Comment:
This is more robust against the `wire` getting dropped (something that seems pretty likely to happen when no more requests are being made).